### PR TITLE
Add review page

### PR DESCRIPTION
**Pull request by meoriordan**
Definition of done: 
Logged out user will be redirected to search page when accessing courses/add_review/
Logged in user will see the form to add a review when accessing courses/add_review/
Logged in user can fill out form 
-if course id and professor name match to an existing class, and all fields are filled out review is created and user sees message that their review was created successfully and an option to see all their reviews on their profile page
-if there was an issue with creating the review, user will see a message that their review was not created

### CODE REVIEW

File: ratenyu/courses/views.py [hunk 1]
```diff
@@ -11,13 +18,52 @@ def course_detail(request: HttpRequest, course_id: str):
...
         reviews_list = create_review_objects(classes)
         reviews_avg = calculate_rating_avg(reviews_list)
-        context = {
-            "classes": classes,
-            "course": course,
-            "reviews_list": reviews_list,
-            "reviews_avg": reviews_avg,
-            "professors_list": professors_list,
-        }
+        context = {"classes": classes, "course": course, "reviews_list": reviews_list}
```
Comment (virenmparmar) [inline ratenyu/courses/views.py:21]: There is an impact of this change on course detail page, list of professor and average review is not visible.

image

File: ratenyu/courses/urls.py [hunk 2]
```diff
@@ -4,5 +4,7 @@
 
 app_name = "courses"
 urlpatterns = [
+    path("add_review/", views.add_review, name="add_review"),
```
Comment (virenmparmar) [inline ratenyu/courses/urls.py:7]: Just a question. Is there any particular reason there are two URLs?

**Review by virenmparmar — CHANGES_REQUESTED** [review]
Everything looks good to me. 
I have added inline comments. 
Just one thing i found. A change that seems to be causing a bug.

File: ratenyu/courses/views.py (hunk 1 above)
Comment (meoriordan) [reply ratenyu/courses/views.py]: nice catch. just pushed a another commit to fix this

**Review by virenmparmar — APPROVED** [review]
LGTM

**Pull request merged**